### ISSUE TITLE
4 new FMTHIST runs testing dust and sea salt emissions

### ISSUE THREAD

**Issue by adamrher** [NCAR/amwg_dev#612]
**Purpose:**

Change dust emissions factor to Danny's preferred values. And do 3 additional runs with different sea salt emissions factors on top of Danny'r preferred emissions factor.
___

**Description:**

baseline is the Leung dust with frontal waves turned on: https://github.com/NCAR/amwg_dev/issues/611

Case1 = baseline + dust_emis_fact = 2.3 + default seasalt_emis_scale (0.75D0)
Case2 = baseline + dust_emis_fact = 2.3 + seasalt_emis_scale = 1.0
Case3 = baseline + dust_emis_fact = 2.3 + seasalt_emis_scale = 1.25
Case4 = baseline + dust_emis_fact = 2.3 + seasalt_emis_scale = 1.5
___
**Case directory:**
- Locally (if still available):
`/glade/campaign/cesm/cesmdata/cseg/runs/cesm2_0/$CASE`

- On github:
https://github.com/NCAR/amwg_dev/tree/$CASE
___
**Sandbox:**
- Locally (if still available):

cesm3_0_alpha04a tag 

- On github:
https://github.com/ESCOMP/CAM/tree/$TAG
___
**Diagnostics:**
- ADF diags (if available)
[https://webext.cgd.ucar.edu/FMTHIST/$CASE/atm/](https://webext.cgd.ucar.edu/FMTHIST/$CASE/atm/)
___
**Output:**
- Raw output on derecho (if still available)
`/glade/derecho/scratch/hannay/archive/$CASE/`
- CAM Climos: `/glade/campaign/cgd/amp/amwg/climo/$CASE`
___
**Contacts:**
@cecilehannay @JulioTBacmeister @swrneale @tilmes 
___
**Extra details:**

**Comment by cecilehannay** [comment]
@adamrher: in these experiments do I keep `clubb_c8 = 4.95` like in #611.

**Comment by adamrher** [comment]
> @adamrher: in these experiments do I keep clubb_c8 = 4.95 like in https://github.com/NCAR/amwg_dev/issues/611.

Yes, we're going to keep that fixed at 4.95 for these experiments.
